articles/active-directory/develop/tutorial-blazor-server.md

```diff
@@ -34,173 +34,63 @@ In this tutorial:
-}
-
-@code {
-
-    private string userDisplayName;
-    private List<MailMessage> messages = new List<MailMessage>();
-
-    private HttpClient _httpClient;
-
-    protected override async Task OnInitializedAsync()
-    {
-        _httpClient = HttpClientFactory.CreateClient();
-
-
-        // get a token
-        var token = await TokenAcquisitionService.GetAccessTokenForUserAsync(new string[] { "User.Read", "Mail.Read" });
-
-        // make API call
-        _httpClient.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Bearer", token);
-        var dataRequest = await _httpClient.GetAsync("https://graph.microsoft.com/beta/me");
-
-        if (dataRequest.IsSuccessStatusCode)
-        {
-            var userData = System.Text.Json.JsonDocument.Parse(await dataRequest.Content.ReadAsStreamAsync());
-            userDisplayName = userData.RootElement.GetProperty("displayName").GetString();
-        }
-
-        var mailRequest = await _httpClient.GetAsync("https://graph.microsoft.com/beta/me/messages?$select=subject,receivedDateTime,sender&$top=10");
-
-        if (mailRequest.IsSuccessStatusCode)
-        {
-            var mailData = System.Text.Json.JsonDocument.Parse(await mailRequest.Content.ReadAsStreamAsync());
-            var messagesArray = mailData.RootElement.GetProperty("value").EnumerateArray();
-
-            foreach (var m in messagesArray)
-            {
-                var message = new MailMessage();
-                message.Subject = m.GetProperty("subject").GetString();
-                message.Sender = m.GetProperty("sender").GetProperty("emailAddress").GetProperty("address").GetString();
-                message.ReceivedTime = m.GetProperty("receivedDateTime").GetDateTime();
-                messages.Add(message);
-            }
-        }
-    }
-
-    public class MailMessage
-    {
-        public string Subject;
-        public string Sender;
-        public DateTime ReceivedTime;
-    }
-}
+## Optional - Create a development SSL certificate
+
+In order to avoid SSL errors / warning when browsing the running application you can use the following on macOS and Windows to generate a self-signed SSL certificate for use by .Net core.
 
+```dotnetcli
+dotnet dev-certs https --trust
 ```
 
-Launch the app. You’ll notice that you're prompted for the newly added permissions, indicating that everything is working as expected. Now, beyond basic user profile data, the app is requesting access to email data.
+## Run the app
 
-After granting consent, navigate to the "Fetch data" page to read some email.
+In your terminal, run the following command:
+
+```dotnetcli
+dotnet run
+```
 
-:::image type="content" source="./media/tutorial-blazor-server/final-app-2.png" alt-text="Screenshot of the final app. It has a heading that says Hello Nicholas and it shows a list of emails belonging to Nicholas.":::
+Browse to the running web application using the URL outputted by the command line.
```

Comment:
outputted? or output? (base: put, past: put, past participle: put)?

Reply:
This is one of the marvels of English... it can either be output or outputted.... according to my web searches.... outputted reads more correctly to me.